ibm_catalog.json

```diff
@@ -0,0 +1,92 @@
+          "databases",
+          "solution"
+        ],
+        "keywords": [
+          "elasticsearch",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "elasticsearch secure",
+          "database",
+          "nosql"
+        ],
+        "short_description": "Creates and configures an instance of IBM Cloud Databases for Elasticsearch.",
+        "long_description": "This architecture supports creating and configuring an IBM Elasticsearch instance with KMS encryption and Context Based Restriction rules.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/main/images/elasticsearch_icon.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Creates an instance of Databases for Elasticsearch",
+            "description": "Creates an IBM Elasticsearch resource instance on the IBM Cloud environment."
+          },
+          {
+            "title": "Creates Context Based Restriction rules",
+            "description": "Creates and configures Context Based Restriction rules."
+          },
+          {
+            "title": "Supports sutoscaling",
+            "description": "Provides the autoscaling feature"
+          },
+          {
+            "title": "Attaches access tags",
+            "description": "Attach access tags to the IBM Elasticsearch instance."
+          }
+        ],
+        "flavors": [
+          {
+            "label": "Secure",
+            "name": "secure",
+            "install_type": "fullstack",
+            "working_directory": "solutions/secure",
+            "compliance": {
+              "authority": "scc-v3",
+              "profiles": [
+                {
+                  "profile_name": "IBM Cloud Framework for Financial Services",
+                  "profile_version": "1.5.0"
+                }
+              ]},
+            "iam_permissions": [
+              {
+                "role_crns": [
+                  "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                ],
+                "service_name": "databases-for-elasticsearch"
+              }
+            ],
+            "architecture": {
+              "descriptions": "This architecture creates an IBM Elastic Search instance in IBM Cloud environment and supports provisioning of a Resource Group, Context Based Restriction rules and supports autoscaling",
+              "features": [
+                {
+                  "title": "IBM Elasticsearch secure instance on existing IBM Cloud secure environment",
+                  "description": "Yes"
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "IBM solution deployable architecture",
```

Comment:
I think - caption should include Elasticsearch too

Reply:
updated